### ISSUE TITLE
Can I resumse yolo nas training?

### ISSUE THREAD

**Issue by ajaybeeravelli** [Deci-AI/super-gradients#1806]
### 💡 Your Question

I am training yolo nas on custom dataset. training is stopped at 25th epoch where as total epochs are 100. Can i able to restart the training from 25th epoch?

**Comment by shaydeci** [comment]
Yes, simpoly set the training hyperparameter 'resume' to True.
